<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HexEditDialog</class>
 <widget class="QDialog" name="HexEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HexEdit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource.qrc">
    <normaloff>:/icons/images/document-binary.png</normaloff>:/icons/images/document-binary.png</iconset>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>359</width>
      <height>99</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEditUnicode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {border-style: outset; border-width: 1px; border-color: black}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAscii">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {border-style: outset; border-width: 1px; border-color: black}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelAscii">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;ASCII:</string>
   </property>
   <property name="buddy">
    <cstring>lineEditAscii</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="labelUnicode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>59</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;UNICODE:</string>
   </property>
   <property name="buddy">
    <cstring>lineEditUnicode</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="labelHex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Hex:</string>
   </property>
   <property name="buddy">
    <cstring>scrollArea</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAscii2Hex">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>53</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ascii2Hex</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUnicode2Hex">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>103</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Unicode2Hex</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkKeepSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Keep Size</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkEntireBlock">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Entire Block</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditAscii</tabstop>
  <tabstop>btnAscii2Hex</tabstop>
  <tabstop>lineEditUnicode</tabstop>
  <tabstop>btnUnicode2Hex</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>chkKeepSize</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>HexEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>HexEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
